resolwe_bio/docker_images/rnaseq/packages-manual/ercc.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+. /var/cache/build/packages-manual/common.sh
+
+# Table with information about ERCC spike-ins
+# Original file is available at https://tools.thermofisher.com/content/sfs/manuals/cms_095046.txt
+
+download_and_verify \
+    resolwebio \
+    assets \
+    ERCC_table \
+    5151b394332477301851404d37b907cb6f27c9aaa2569e288531a839d6ca2b3b \
+    https://drive.google.com/uc?id=193PpxSoNhFwYUpTMh9nGt44K72tc0F9f \
+    '' \
+    ERCC_table.txt
```

Comment:
Would be nice to have a more informative file name.

Reply:
How do you like `ERCC_spikein_concentrations.txt`?

Reply:
I was thinking more in line of mentioning Thermofisher or the type of kits using this... Just cause there are more and more files on the Google Drive and we need to be able to know what's what. Anyway, it's not super important and I was not sure whether I should point it out at all or not. Your call how you handle it.

Reply:
Let's just leave it for now. Files belonging in Docker images are quite well separated and this is the first on regarding ERCC. If and when we have more of them we should have a better idea how to rename/organize them.